app/spell_check.py

```diff
@@ -1,14 +1,16 @@
 from spellchecker import SpellChecker
 import re
```

Comment:
If we are using regex now, do we still need the "import re" here?

Reply:
Good catch! I was still using re in one place in another function, but for consistency I've updated it to use regex as well.